mina-sshd-api-core/src/test/java/io/jenkins/plugins/mina_sshd_api/core/authenticators/MinaSSHPasswordKeyAuthenticatorTest.java

```diff
@@ -50,7 +49,8 @@ public void tearDown() {
     }
 
     @Before
-    public void setUp() throws IOException {
+    public void setUp() throws Exception {
+        user = new UsernamePasswordCredentialsImpl(CredentialsScope.SYSTEM, null, "foobar", "foobar", "foomanchu");
```

Comment:
Required because of constructor thrown exception in new version of credentials plugin: https://github.com/jenkinsci/credentials-plugin/commit/81ef4269d76477f11c4cbd9fb536de81e3a53060.